modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,209 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function(){
+
+     /* Master Tabs */
+
+     // if no selected => select first
+     if ($('.master-tabs a[aria-selected]').length == 0) {
+         $('.master-tabs a:first').attr('aria-selected', 'true');
```

Comment:
@ayumihamsaki could you refactor it to a single line, like so `.attr({ aria-selected:'true', tabindex:'0' })`

Reply:
Fixed.